neoforge/src/main/resources/architectury.mixins.json

```diff
@@ -5,15 +5,16 @@
   "compatibilityLevel": "JAVA_16",
   "minVersion": "0.8",
   "client": [
-    "client.MixinCommandSourceStack",
```

Comment:
Maybe don't induce unnecessary changes (would complicate future merges), also applys to the reordering of MixinChunkSerializer below